Proteomics/ProteolyticDigestion/PeptideWithSetModifications.cs

```diff
@@ -59,14 +61,19 @@ public double MonoisotopicMass
                     _monoisotopicMass = WaterMonoisotopicMass;
                     foreach (var mod in AllModsOneIsNterminus.Values)
                     {
-                        _monoisotopicMass += mod.monoisotopicMass;
+                        _monoisotopicMass += mod.MonoisotopicMass;
                     }
                     _monoisotopicMass += BaseSequence.Select(b => Residue.ResidueMonoisotopicMass[b]).Sum();
                 }
-                return _monoisotopicMass.Value;
+                return RoundedDouble(_monoisotopicMass.Value);//rounding this at 1e-9 tolerance
             }
         }
 
+        public double RoundedDouble(double myNumber)
```

Comment:
delete this, move the other method to a more accessible location, and call that method. we don't want duplicate methods doing the same work

Reply:
same as before. do we have a utility class?